app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -270,13 +323,58 @@ private void addBuildingMarkersAndPolygons() {
         map.setOnPolygonClickListener(new GoogleMap.OnPolygonClickListener() {
             @Override
             public void onPolygonClick(Polygon polygon) {
+
+                /**
+                 * ONLY FOR DEMO PURPOSES
+                 */
+                Building building = Campus.SGW.getBuilding(polygon);
+                if(building == null){
+                    building = Campus.LOY.getBuilding(polygon);
+                }
+                ((MainActivity) getActivity()).createToast(building.getShortName());
+                String buildingName = building.getShortName();
+                buildingInfoFragment.setBuildingInformation(buildingName, "add", "7:00", "23:00");
+                buildingInfoFragment.clear();
+                // TEMPORARY
```

Comment:
Sure I can cahnge it. Again i didn't write this. I was told to do it this way but i'll change it